server/views/pages/overview/partials/supportNeedsTabContents.test.ts

```diff
@@ -25,57 +25,38 @@ describe('Support Needs tab view', () => {
 
     // When
     const $ = cheerio.load(compiledTemplate.render(viewContext))
+    setupCheerioExtensionFunctions($)
 
     // Then
     const healthAndSupportNeedsCard = $('#health-and-support-needs-summary-card')
     // expect there to be only 1 gov-uk-summmary-list representing the data from the 1 prison in aValidPrisonerSupportNeeds()
     expect(healthAndSupportNeedsCard.find('.govuk-summary-list').length).toEqual(1)
-    expect(
-      healthAndSupportNeedsCard //
-        .find('h3')
-        .text(),
-    ).toContain(`Jimmy Lightfingers's learning difficulties, disabilities and health needs recorded whilst at`)
-    expect(
-      healthAndSupportNeedsCard //
-        .find('h3')
-        .text(),
-    ).toContain('MOORLAND (HMP & YOI)')
-    expect(
-      healthAndSupportNeedsCard //
-        .find(`.govuk-summary-list__key:contains('Rapid assessment')`)
-        .next()
-        .text()
-        .trim(),
-    ).toEqual('18 February 2022')
-    expect(
-      healthAndSupportNeedsCard //
-        .find(`.govuk-summary-list__key:contains('In-depth assessment')`)
-        .next()
-        .text()
-        .trim(),
-    ).toEqual('Not recorded in Curious')
-    expect(
-      healthAndSupportNeedsCard //
-        .find(`.govuk-summary-list__key:contains('Primary LDD and health needs')`)
-        .next()
-        .text()
-        .trim(),
-    ).toEqual('Visual impairment')
-    expect(
-      healthAndSupportNeedsCard //
-        .find(`.govuk-summary-list__key:contains('Additional LDD and health needs')`)
-        .next()
-        .find('li')
-        .toArray()
-        .map(el => $(el).text()),
-    ).toEqual(['Hearing impairment', 'Mental health difficulty', 'Social and emotional difficulties'])
+    expect(healthAndSupportNeedsCard.heading()).toContain(
```

Comment:
I've found out how to extend cheerio - https://cheerio.js.org/docs/advanced/extending-cheerio

(@mattwills-moj-digital  and I would know this type of approach as extension functions 😁 )